tests/rptest/tests/group_membership_test.py

```diff
@@ -113,3 +116,187 @@ def _check_no_duplicates():
                    10,
                    backoff_sec=2,
                    err_msg="found persistent duplicates in groups listing")
+
+
+class GroupMetricsTest(RedpandaTest):
+    def __init__(self, ctx, *args, **kwargs):
+
+        extra_rp_conf = dict(default_topic_replications=3, )
+
+        super(GroupMetricsTest, self).__init__(test_context=ctx,
+                                               num_brokers=3,
+                                               extra_rp_conf=extra_rp_conf)
+        self._ctx = ctx
+        random.seed(42)
```

Comment:
I'd remove this unless there is a specific reason to leave it in in which case there should be a note associated with it.